index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="contact.css">
+    <title>ContactUs</title>
+</head>
+
+<body>
+    <section>
+        <div class="container">
+            <div class="contactinfo">
+                <div>
+                    <h2>Contact Us</h2>
+                    <ul class="info">
+                        <li>
+                            <span><img src="images/location.png" alt=""></span>
```

Comment:
Alt attribute value?

Reply:
actually, all images are available
that's why I thought there is no need to give alt value

Reply:
its is a good practice to provide ```alt``` value. it is good you are able to see, if xyz user is not able to see the image then?

Reply:
i updated the ALT values